packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -72,18 +72,38 @@ type Labels = {|
 // values to the parent components that are instantiating this component
 // @see https://flow.org/en/docs/react/hoc/#toc-exporting-wrapped-components
 type DefaultProps = {|
-    /**
-     * An index that represents the index of the focused element when the menu
-     * is opened.
-     */
-    initialFocusedIndex?: number,
-
     /**
      * Whether this menu should be left-aligned or right-aligned with the
      * opener component. Defaults to left-aligned.
      */
     alignment: "left" | "right",
 
+    /**
+     * Whether to auto focus an option.
+     */
+    autoFocus: boolean,
```

Comment:
**suggestion (non-blocking):** Could you include what the default value is in the docstring as well as when people my want to use the non-default value?